### PR TITLE
Added sub comment to method name handler

### PR DESCRIPTION
**Pull request by hitautodestruct**

### CODE REVIEW

**Comment by phanan** [comment]
Please fix the merge conflict.

File: src/v2/api/index.md [hunk 1]
```diff
@@ -674,14 +674,15 @@ type: api
...
-      // deep watcher
+      // the callback will be called whenever any of the watched object properties change regardless of their nested depth
       c: {
         handler: function (val, oldVal) { /* ... */ },
         deep: true
       },
       // the callback will be called immediately after the start of the observation
       d: {
-        handler: function (val, oldVal) { /* ... */ },
+        // use a method name, instead of an anonymous function
```
Comment (Justineo) [inline src/v2/api/index.md]: This looks a bit misleading that people may assume using method name here is required. Maybe we can use something like:

```suggestion
        // string method name also works here
```

Comment (phanan) [reply src/v2/api/index.md]: Or "a method name also works here" because "name" already implies a string.

**Comment by sdras** [comment]
This has no movement for a while and is stuck in merge conflict since April. I'm closing it out, please feel free to do a fresh PR

**Pull request closed without merging**